101-functions-managed-identity/azuredeploy.json

```diff
@@ -0,0 +1,286 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "functionAppName": {
+            "type": "string",
+            "metadata": {
```

Comment:
since this is globally unique, add a defaultValue using uniqueString, e.g.

"defaultValue": "[uniqueString(resourceGroup().id)]"